dashboard/package.json

```diff
@@ -40,10 +46,13 @@
   "scripts": {
     "build": "react-app-rewired build",
     "doc": "jsdoc -c jsdoc.json",
+    "dev": "npm run server & npm start",
```

Comment:
Alphabetically, `doc` and `dev` are reversed.

Reply:
Updated the order :smile: